plugins/woocommerce/src/Blocks/Package.php

```diff
@@ -66,9 +66,11 @@ public static function get_path() {
 	 * Returns an instance of the FeatureGating class.
 	 *
 	 * @return FeatureGating
+	 * @deprecated since 9.6.0, use wp_get_environment_type() instead.
 	 */
 	public static function feature() {
-		return self::get_package()->feature();
+		wc_deprecated_function( 'Package::feature', '9.6.0', 'wp_get_environment_type' );
```

Comment:
Should it be `9.6`?

```suggestion
		wc_deprecated_function( 'Package::feature', '9.6', 'wp_get_environment_type' );
```

Reply:
Looks like we use both, but addressed here: https://github.com/woocommerce/woocommerce/pull/52768/commits/b0c26654e3467e0bb91fcf1eabc34b99d2bdbf03